var/spack/repos/builtin/packages/graphviz/package.py

```diff
@@ -59,7 +62,9 @@ def install(self, spec, prefix):
         #       include <X11/Xlib.h>
         if sys.platform == 'darwin':
             options.append('CFLAGS=-I/opt/X11/include')
+        options.append('--with-ltdl-lib=%s/lib' % self.spec['libtool'].prefix)
 
-        configure(*options)
-        make()
-        make("install")
+        # A hack to patch config.guess in the libltdl sub directory
+        shutil.copyfile('./config/config.guess', 'libltdl/config/config.guess')
```

Comment:
Is there any way we can work around this hack from `AutotoolsPackage.do_patch_config_guess`? Maybe we just need to patch all `config.guess` recursively?